galasa-parent/dev.galasa/src/main/java/dev/galasa/api/run/Run.java

```diff
@@ -24,15 +22,15 @@ public class Run {
     private String  stream;
     private String  repo;
     private String  obr;
-    private Boolean local;
-    private Boolean trace;
+    private boolean local;
+    private boolean trace;
     private String  rasRunId;
     
     public Run() {};
 
     public Run(String name, Instant heartbeat, String type, String group, String test, String bundleName,
             String testName, String status, String result, Instant queued, Instant finished, Instant waitUntil,
-            String requestor, String stream, String repo, String obr, Boolean local, Boolean trace, String rasRunId) {
+            String requestor, String stream, String repo, String obr, boolean local, boolean trace, String rasRunId) {
```

Comment:
This is an external interface changed in our galasa api...
Need to deprecate the old constructor and add a new one, and the old one should call the new one ?